app/src/main/resources/org/jboss/hal/client/configuration/subsystem/transactions/TransactionView.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/hal/hal.next/develop/spi/src/main/resources/org/jboss/hal/spi/MbuiView.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<!--
+  ~ Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<view>
+    <vertical-navigation id="transaction-vertical-navigation">
+
+        
+        <item id="attributes-config-item" title="Configuration" icon="fa fa-list-ul">
+            <metadata address="/{selected.profile}/subsystem=transactions">
+                <h1>Transaction Manager</h1>
+                <p>{{metadata.getDescription().getDescription()}}</p>
+                <form id="attributes-form" title="Transaction Manager" auto-save="true">
+                    <attributes>
+                        <attribute name="default-timeout" />
+                        <attribute name="enable-tsm-status" />
+                        <attribute name="journal-store-enable-async-io" />
+                        <attribute name="jts" />
+                        <attribute name="node-identifier" />
+                        <attribute name="statistics-enabled" />
+                        <attribute name="use-journal-store" />
+                    </attributes>
+                </form>
+            </metadata>
+        </item>
+
+        <item id="process-item" title="Process" icon="fa fa-clipboard">
+            <metadata address="/{selected.profile}/subsystem=transactions">
+                <h1>Process ID</h1>
+                <p>{{metadata.getDescription().getDescription()}}</p>
+                <form id="process-form" title="Process ID" auto-save="true">
```

Comment:
The current `TransactionPresenter` has a special handling when switching between `process-id-uuid` and `process-id-socket-binding` (see [HAL-435](https://issues.jboss.org/browse/HAL-435)). 

So `auto-save="true"` should be changed to something like `on-save="{{presenter.saveProcessSettings(form, changedValues)}}"`. It should be possible to migrate the existing `saveProcessSettings()` [code](https://github.com/hal/core/blob/master/gui/src/main/java/org/jboss/as/console/client/shared/subsys/tx/TransactionPresenter.java#L150) with minor modifications.